pkg/api/v1alpha1/cluster_types.go

```diff
@@ -51,7 +51,8 @@ type ClusterSpec struct {
 	// Sysctl properties to be applied during initialization
 	// given as a list of key=value pairs.
 	// Example: fs.aio-max-nr=232323
-	Sysctls []string `json:"sysctls,omitempty"`
+	Sysctls    []string `json:"sysctls,omitempty"`
+	ScyllaArgs *string  `json:"scyllaArgs,omitempty"`
```

Comment:
It would be better to have it as Map, but unfortunatelly controller-get v0.3.0 does not support nillable values in the map, whcih does not allow us to have flag arguments there.
Currently I've move it back to string.